### ISSUE TITLE
[TW-1786] Project name can't start with 'project'

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1809]
_Dmitriy Bogdanov on 2016-03-21T11:04:06Z says:_ 

 An error is reported if a project name starts with "project":
{code}
$ task add project:projecttest Test
The expression could not be evaluated.
{code}
It's also happens with `modify` command
{code}
$ task 1 modify project:project_test
The expression could not be evaluated.
{code}

But it is possible to add a task with a project named "project".
{code}
$ task add project:project Test
Created task 2.
The project 'project' has changed.  Project 'project' is 0% complete (1 task remaining).
{code}

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2016-03-21T11:04:06Z
Modified: 2017-01-24T17:09:59Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2016-03-21T13:11:28Z says:_ 

 That's crazy.  Thank you!

**Comment by regnat** [comment]
This isn't specific to project (neither the field, neither the name):

```
$ task add bar
Created task 1.
$ task add tags:priority_foo bar
The expression could not be evaluated.
$ task add tags:tagss bar
The expression could not be evaluated.
$ task add description:projectdescription
The expression could not be evaluated.
```

(But in the case of tags, interestingly enough, this works:
```
$ task add +priority_foo bar
Created task 2.
```
)

**Comment by tbabej** [comment]
This is fixed on 2.5.2 and 2.6.0 (both original reporter's and @regnat's examples work).
